.flake8

```diff
@@ -0,0 +1,9 @@
+# Generated by synthtool. DO NOT EDIT!
```

Comment:
Is this actually generated by synthtool? Let's add a comment with a link to the template file if not.

Reply:
This was not copied in using the template. Removed the comment.